taxonomy-editor-frontend/src/pages/editentry/ListAllOtherProperties.jsx

```diff
@@ -0,0 +1,27 @@
+import { Box, Typography } from "@mui/material";
+
+// Parent component used for rendering info
+// on a stopword, synonym, header or footer
+
+const ListAllOtherProperties = ({ nodeObject, id, setNodeObject }) => {
+
+    return ( 
+        <Box className="all-node-properties">
+            {/* Comments */}
+            <Typography sx={{ml: 4, mt: 2, mb: 1}} variant='h5'>Comments</Typography>
+            
+            {/* Stopwords or Synonyms */}
+            <div className="tags">
+                {id.startsWith('stopword') ?
+                    <Typography sx={{ml: 4, mt: 1, mb: 1}} variant='h5'>
```

Comment:
from what happens in the parent component, this id could be header, footer, synonym and stopword, not only stopword or synonyms, if you only care about those two cases please add a check

Reply:
The check will be coming in future PR, where the length of properties to be rendered is validated. Currently, this will work for the header/footer case also.